py3status/modules/gitlab.py

```diff
@@ -17,7 +17,7 @@
     thresholds: specify color thresholds to use (default [])
 
 Format placeholders:
-    See `sp` below for an explicit list of supported GitLab placeholders to
+    See `sp` and `pipe` below for an explicit list of supported GitLab placeholders to
```

Comment:
This is too long. Can you change it to...
```
    See `sp` below for a full list of supported GitLab placeholders
    to use. Not all of GitLab placeholders will be usable.
```

Don't add `and pipe` because I only see you added `pipelines_status` placeholder. This is for `single_project` 50+ placeholders (flattened) https://docs.gitlab.com/ee/api/projects.html#get-single-project. Few examples...
* `{merge_requests_enabled}`
* `{last_activity_at}`
* `{archived}`
* `{statistics_repository_size}`
* and more...

Don't black this module... It's unreviewable now :sob: Also, the owner will do it again.

Reply:
I read too quick between two 🍾 🥂 , Ok I rollback black :)

Reply:
Fixed.